### PR TITLE
Adds Vulpkanin EVA suit sprite

### PR DESCRIPTION
**Pull request by Jovaniph**
This PR adds the EVA suit sprite for Vulpkanin when they put on the space suit. All it does is change the sprite for vulpkanin. It is not an additional suit in the game like the Vox's EVA suit.

img1
img3
img2

:cl: Jovaniph
rscadd: Adds Vulpkanin EVA suit sprite
/:cl:

### CODE REVIEW

**Comment by Anasari** [comment]
Looks great.

**Comment by Jovaniph** [comment]
ohh..there is two pixels on the damn shoe... part..gotta fix that o3o

**Comment by tristan1333** [comment]
It looks like fluffy metal which kinda doesn't make sense but uhh yeah I mean if we really need it.

**Comment by Xantholne** [comment]
Well, there isn't one, so for vulps wearing a soft suit means 

1) Ears stick out awkwardly as the helmet forms around for a human based head which doesn't make sense

2) The muzzle on a vulp also sticks out with the mask which looks incredibly awkward and has obvious clipping issues with the blue face guard

3) Tails literally disappear into nothing, they just vanish into a perfect human body which looks not so well and very unrealistic

**Comment by tristan1333** [comment]
I mean tucking the tails in kinda does make sense. Rest looks fine but the tail is errr yeah looks dumb to me. Fluffy metal? Metal tail?

**Comment by Fox-McCloud** [comment]
Conflicts.

**Comment by Jovaniph** [comment]
Code retested and conflict resolved.

**Comment by Rumiluntti** [comment]
My hungover mind made laugh at the tail that looks like it's stuffed in a garbage bag.

**Comment by tristan1333** [comment]
@Rumiluntti But the bag is fluffy

**Comment by Dinarzad** [comment]
These are normal EVA suits which are supposed softsuits, not with any sort of metal armor.
Plus making races stand out a little more in the everyday is nice, even the baseline/non-karma ones.

**Comment by FlattestGuitar** [comment]
Anomaly and Paramedic suits inherit the sprites. They won't work right.

https://github.com/ParadiseSS13/Paradise/pull/6747 accounts for that.

**Comment by Jovaniph** [comment]
uggh...

File: code/modules/research/xenoarchaeology/tools/anomaly_suit.dm [hunk 1]
```diff
@@ -23,10 +23,22 @@
 	item_state = "cespace_suit"
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 100, rad = 100)
 	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank)
+	+	sprite_sheets = list(
```
Comment (Jovaniph) [inline code/modules/research/xenoarchaeology/tools/anomaly_suit.dm]: uggh.

File: code/modules/research/xenoarchaeology/tools/anomaly_suit.dm [hunk 2]
```diff
@@ -23,10 +23,16 @@
...
 
 /obj/item/clothing/head/helmet/space/eva/anomaly
 	name = "Excavation hood"
 	desc = "A pressure resistant excavation hood partially capable of insulating against exotic alien energies."
 	icon_state = "cespace_helmet"
 	item_state = "cespace_helmet"
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 100, rad = 100)
+	species_fit = null
+	sprite_sheets = null
+	sprite_sheets_obj = null
```
Comment (Jovaniph) [inline code/modules/research/xenoarchaeology/tools/anomaly_suit.dm]: There is no sprite for either the vulp or vox for these. So these are null for a reason.

**Comment by tristan1333** [comment]
eh I guess. The grey tail just looks dumb imo. BUT if no one else thinks that yeah ignore me.

**Comment by Jovaniph** [comment]
@tristan1333 if someone is willing to retexture (change it, w/e), they can do so. It can always be changed later.

**Comment by Xantholne** [comment]
I mean what else is the tail going to be repalced with? The suit is grey, the tail will be grey.

**Pull request merged**